kong/conf_loader.lua

```diff
@@ -294,6 +295,22 @@ local function check_and_infer(conf)
     end
   end
 
+  local mask = 0
+  if conf.tokens then
+    local allowed = constants.HEADER_MASKS
+    for _, token in ipairs(conf.tokens) do
+      if token == "off" then
+          mask = 0
+          break
+      elseif allowed[token] == nil then -- TODO what about case sensetive??
+        errors[#errors+1] = "tokens: invalid entry '" .. tostring(token) .. "'"
+      else
```

Comment:
style: ditto (line jump)

Reply:
Sorry about all the styling issues in this (and all the previous) PR. I need to figure out my styling and code convention issues.